nncf/common/graph/graph.py

```diff
@@ -68,8 +82,8 @@ def node_name(self) -> NNCFNodeName:
         return cast(NNCFNodeName, self._attributes[NNCFNode.NODE_NAME_ATTR])
 
     @property
-    def metatype(self) -> Type[OperatorMetatype]:
-        return cast(Type[OperatorMetatype], self._attributes[NNCFNode.METATYPE_ATTR])
+    def metatype(self) -> OperatorMetatype:
```

Comment:
Could you briefly explain of these changes. I think, that the previous version is correct.

Reply:
reverted